semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Settings.scala

```diff
@@ -1,31 +1,56 @@
 package scala.meta.internal.metacp
 
+import java.io.PrintStream
+
+import org.langmeta.io.AbsolutePath
+import org.langmeta.io.Classpath
+
+import scala.meta.cli.Metacp
+
 final case class Settings(
-    cps: List[String] = Nil,
-    d: String = "."
-)
+    out: PrintStream = System.out,
+    err: PrintStream = System.err,
```

Comment:
Whatever we decide here, I think we should do exactly the same for `metap` and `metac` for consistency reasons.